tools/tabular_to_fasta/tabular_to_fasta.py

```diff
@@ -13,53 +13,56 @@ def stop_err(msg):
 
 
 def __main__():
-    infile = sys.argv[1]
-    title_col = sys.argv[2]
-    seq_col = sys.argv[3]
-    outfile = sys.argv[4]
+    parser = argparse.ArgumentParser()
+    parser.add_argument('input_file', type=argparse.FileType())
+    parser.add_argument('title_col')
+    parser.add_argument('seq_col')
+    parser.add_argument('output_file', type=argparse.FileType('w'))
+    args = parser.parse_args()
+    title_col = args.title_col
+    seq_col = args.seq_col
 
-    if title_col == None or title_col == 'None' or seq_col == None or seq_col == 'None':
+    if title_col is None or title_col == 'None' or seq_col is None or seq_col == 'None':
         stop_err("Columns not specified.")
     try:
         seq_col = int(seq_col) - 1
-    except:
+    except ValueError:
         stop_err("Invalid Sequence Column: %s." % str(seq_col))
 
-    title_col_list = title_col.split(',')
+    title_col_list = title_col.strip().split(',')
```

Comment:
Given the chat with Marius on Matrix, revert this?

```suggestion
    title_col_list = title_col.split(',')
```